src/components/standalone/interfaces_and_devices/ConfigureDeviceDrawer.vue

```diff
@@ -922,12 +990,24 @@ async function listZonesForDeviceConfig() {
         <div v-show="['static', 'bonding'].includes(protocol)" class="space-y-6">
           <!-- ipv4 address -->
           <NeTextInput
-            :label="t('standalone.interfaces_and_devices.ipv4_address_cidr')"
+            :label="
+              logicalIfaceType === 'bond'
+                ? t('standalone.interfaces_and_devices.management_ipv4_address_cidr')
+                : t('standalone.interfaces_and_devices.ipv4_address_cidr')
+            "
             v-model.trim="ipv4Address"
             :invalidMessage="t(error.ipv4Address)"
             :disabled="loading.configure"
```

Comment:
Why not disabling it to avoid errors from the user?
```suggestion
          :disabled="loading.configure || logicalIfaceType === 'bond'"
```

Reply:
Haven't thought of that. Nice.